spec/hydra/works/models/concerns/generic_file/characterization_spec.rb

```diff
@@ -0,0 +1,182 @@
+require 'spec_helper'
+
+# Mocking out the Hydra::Works::GenericFile sufficiently to add original_file
```

Comment:
Initially, I did try that.  the `mock_add_file_to_generic_file` function threw errors when I put it in the spec_helper.